go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/google/go-cmp v0.5.9
 	github.com/google/uuid v1.3.1
 	github.com/hashicorp/go-azure-helpers v0.59.0
-	github.com/hashicorp/go-azure-sdk v0.20230907.1113401
+	github.com/hashicorp/go-azure-sdk v0.20230911.1163300
```

Comment:
For future reference these have to go into their own PR to avoid conflicts. So can you please revert this and also do a rebase since the `go-azure-sdk` has already been updated since the opening of this PR.